mkdocs.yml

```diff
@@ -62,6 +62,7 @@ nav:
             - Connect to a network:
                 - Connect to Mainnet: HowTo/Get-Started/Connect/Connect-To-Mainnet.md
                 - Connect to a testnet: HowTo/Get-Started/Connect/Connect-To-Testnet.md
+            - Start Teku from a recent state: HowTo/Get-Started/Snapshot-Start.md
```

Comment:
Have we settled on "snapshot sync"? I kind of prefer "checkpoint sync" since it seems more descriptive.

Reply:
@bgravenorst ?

Reply:
Yep will update this. I used checkpoint throughout the content. But I created this file before giving much thought to naming.